package.json

```diff
@@ -26,85 +26,27 @@
         "default": "./esm/index.mjs"
       },
       "module": {
-        "types": "./index.d.ts",
+        "types": "./esm/index.d.ts",
```

Comment:
Is this change the key to fix the issue? I mean the file content must be the same, so the file path was the root cause, right? Why wasn't it an issue before #2157?

It feels like reverting #2157 should be appropriate, if we **don't know** why it caused such an issue.

Reply:
@dai-shi btw, I updated the exports to match jotai's and valtio's exports

Reply:
@dai-shi but we can revert the changes

Reply:
Yeah > https://github.com/pmndrs/zustand/pull/2170#discussion_r1379421218